### ISSUE TITLE
Alter code extracting naming convention in combine_psm_fractions

### ISSUE THREAD

**Issue by baynec2** [baynec2/GLabR#2]
Right now the function works by splitting the file name by _ into the sample name and fraction. We will also want to retain the DG number. Need to update both the code and lab naming conventions to work appropriately. Will need to think some more about the best way to do this.

**Comment by c6sanche** [comment]
Maybe think of splitting it my the File ID column instead? I think sometimes it changes often with each PD version

**Comment by baynec2** [comment]
Do you know how the FileID column works? Does F1.9 mean Plex1.fraction9? If you had three plexes would it be F1.1-12, F2.1-12, and F3.1-12?

**Comment by c6sanche** [comment]
Yup that's how it works right now. It based on how we load our data into Proteome Discoverer when setting up a search though. So people would have to pay careful attention to load their first plex as F1, there second as F2, etc so that it's not confusing after the fact when you are trying to assign your columns to the correct samples
